apps/public-docsite-v9/.storybook/preview.js

```diff
@@ -47,20 +47,14 @@ export const parameters = {
           'Recipes',
         ],
         'Theme',
-        [
-          'Border Radii',
-          'Colors',
-          'Fonts',
-          'Motion',
-          'Shadows',
-          'Spacing',
-          'Stroke Widths',
-          'Typography',
-          'Theme Designer',
-        ],
+        ['Border Radii', 'Colors', 'Fonts', 'Shadows', 'Spacing', 'Stroke Widths', 'Typography', 'Theme Designer'],
         'Components',
+        'Compat Components',
+        'Preview Components',
         'Icons',
         ['Overview', 'Catalog'],
+        'Motion',
```

Comment:
I wonder if `Motion` and `Utilities` should actually move up the list? `Components` is a large section that effectively hides anything that comes after it. Of course, people are looking for `Components` but perhaps this is also a reason it should come later -- people aren't necessarily looking for `Motion` and `Utilities` so we should elevate them rather than hide them because people _are_ looking for `Components`.

Reply:
@spmonahan I was thinking about that, too. I will iterate later on some other parts of docs and I think that I will apply the suggested order.